packages/asc-ui/src/components/Menu/MenuButton/MenuButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import MenuButtonStyle, {
+  MenuButtonTextStyle,
+  MenuButtonTextWrapperStyle,
+  Props,
+} from './MenuButtonStyle'
+
+const MenuButton: React.FC<
+  Props &
+    React.ButtonHTMLAttributes<HTMLButtonElement> &
+    React.AnchorHTMLAttributes<HTMLAnchorElement>
+> = ({ children, ...otherProps }) => (
+  <MenuButtonStyle iconSize={14} variant="blank" {...otherProps}>
```

Comment:
Do we export the variants somewhere as constants?

Reply:
not now, have to figure out how to do this in combination with enums... leave it out of scope for now